### DISCUSSION TITLE
크롬 확장 한컴독스 로그인 불가 현상

### DISCUSSION THREAD

**Discussion by loquenahak** [List-KR/community#48] [AdGuard]
애드가드 필터 이슈 페이지에도 남겼습니다만, 카테고리가 애매해서 중복 신고 드립니다. (https://github.com/AdguardTeam/AdguardFilters/issues/166547#issuecomment-1826884259)

크롬 확장 프로그램인 한컴독스 Hancom docs 가 애드가드 활성화시 로그인 및 로그인 유지가 불가능하여 신고드립니다.

자세한 사항은 애드가드 필터 이슈 페이지 참고 부탁드립니다.

고맙습니다.

**Comment by piquark6046** [comment]
오늘 확인해보겠습니다.

**Reply by piquark6046** [reply]
고쳤습니다. 업데이트 후에 잘 되는 지 확인해주세요. https://github.com/AdguardTeam/AdguardFilters/commit/b83317fc425f6b2ca6be5f8caefe716793edc1cd

**Comment by loquenahak** [comment]
> 고쳤습니다. 업데이트 후에 잘 되는 지 확인해주세요. [AdguardTeam/AdguardFilters@b83317f](https://github.com/AdguardTeam/AdguardFilters/commit/b83317fc425f6b2ca6be5f8caefe716793edc1cd)

고맙습니다. 업데이트 후에 잘 되다가, 최근 며칠 사이에 다시 동일한 증상이 일어납니다. 비단 한컴독스 뿐만 아니라, 다른 웹사이트들도 약간씩 문제가 있는 것 같은데 크롬 업데이트 후 문제가 생긴 것 같기도 하고요.
